Mathlib/Mathport/Syntax.lean

```diff
@@ -164,6 +171,9 @@ namespace Tactic
 /- M -/ syntax (name := congrM) "congrm " term : tactic
 /- E -/ syntax (name := acChange) "ac_change " term (" using " num)? : tactic
 
+/- S -/ syntax (name := rcases?) "rcases?" casesTarget,* (" : " num)? : tactic
+/- S -/ syntax (name := rintro?) "rintro?" (" : " num)? : tactic
```

Comment:
Haha, well spotted.  That's what you get for grepping for `elab.*"rcases\?"` and missing the TODO comment around it. :smile: